RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/Encoders.kt

```diff
@@ -23,7 +23,7 @@ sealed interface Encoder {
 class RawEncoder(private val m: DcMotorEx) : Encoder {
     override var direction: DcMotorSimple.Direction = DcMotorSimple.Direction.FORWARD
 
-    private fun applyDirection(x: Int): Int {
+    private fun applyDirection(x: Double): Double {
```

Comment:
I'm not really sure what a fractional direction means. Are you trying to use this for a more generic scale? Similar to the position/velocity container, I'm not sure that's what this class is for.

Reply:
As I understood this, x was the input position or velocity that the direction should be applied to, rather then the direction itself? Or are you referring to something else?

Reply:
Ah my bad you're totally right 😅